index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <div>
+        <header class="header flex-row">
+            <div class="menu-button">
+                <a class="logo">JEPM</a>
+            </div>
+            <nav class="menu-mobile">
+                <button type="button" class="menu-button"><img src="./img/header-menu.png" alt="menu-image"></button>
+            </nav>
+        </header>
```

Comment:
- [ ] As required for the project, to match the Figma design, I think it would be better if you improve again your Header design to stick as much as possible to the one from your chosen template. e.g.: `Fonts(weight, size)`, `line height`, `spacing at left and right`,...
### Figma design
![header](https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/toolbar-mobile.png)
 
 ### Your Design
![Screenshot from 2023-04-27 06-17-22](https://user-images.githubusercontent.com/83042742/234758721-3b77b0df-fad4-4f05-8fce-ca4af54c8fec.png)